### PR TITLE
Refactored CSS in src/screens/OrganizationEvents/OrganizationEvents.tsx

### LINKED ISSUE #2888: Refactor CSS in `src/screens/OrganizationEvents/OrganizationEvents.tsx`
# Refactor CSS in this file

To improve the UI/UX of Talawa-Admin we have been working on:
1. making the application more suitable to color blind users
2. streamlining all CSS into a single global file

This file contains embedded CSS references that should be completely moved to:
- `src/style/app.module.css`

## Additional information:
1. The foundational work for this was completed in this PR:
    - https://github.com/PalisadoesFoundation/talawa-admin/pull/2466
2. The desired single CSS file is:
    - `src/style/app.module.css`

## The challenge:
The goal is for this file to use this new design pattern. This means that:
1. All embedded CSS in this file must be merged into the global CSS file using pre-existing CSS.
1. All embedded CSS in this file must be removed

### PR DESCRIPTION
**Pull request by aadhil2k4**
**What kind of change does this PR introduce?**

Refactoring

**Issue Number:**

Fixes #2888 

**Did you add tests for your changes?**

No, only CSS changes

**Snapshots/Videos:**

N/A, visual output remains unchanged

**Does this PR introduce a breaking change?**

No

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

Yes

## Summary by CodeRabbit

- **New Features**
	- Introduced new styling classes for organization events, enhancing layout and usability.
	- Added responsive design adjustments for various components.

- **Bug Fixes**
	- Removed outdated styles affecting the organization events interface.

- **Refactor**
	- Updated class names to follow a new naming convention for better organization and clarity.
	- Changed the source of styles to a centralized stylesheet for improved maintainability.

### CODE REVIEW

**Comment by palisadoes** [comment]
Please make coderabbit.ai approve your changes

**Comment by palisadoes** [comment]
Please fix the conflicting file

**Comment by palisadoes** [comment]
Please make code rabbit approve your changes

**Comment by palisadoes** [comment]
Please fix the failing tests. Add tests to make sure the code quality improves

**Comment by aadhil2k4** [comment]
@palisadoes this is the fix that I am being suggested for the failing test 
image

Should I proceed with it?

**Comment by palisadoes** [comment]
1. Merge your local branch with the latest upstream and push to your origin.
2. That should clear the error
3. We added some new checks after you initially submitted your PR

File: src/style/app.module.css [hunk 1]
```diff
@@ -944,6 +930,96 @@ hr {
...
 }
 
+.justifyspOrganizationEvents {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 20px;
+}
+
+.titlemodalOrganizationEvents {
+  color: #707070;
```
Comment (palisadoes) [inline src/style/app.module.css:940]: Please change this to a variable. The color of something similar may already be used

File: src/style/app.module.css [hunk 2]
```diff
@@ -944,6 +930,96 @@ hr {
...
+  font-weight: 600;
+  font-size: 20px;
+  margin-bottom: 20px;
+  padding-bottom: 5px;
+  border-bottom: 3px solid #eaebef;
```
Comment (palisadoes) [inline src/style/app.module.css:945]: Please change this to a variable. The color of something similar may already be used

**Review by palisadoes — CHANGES_REQUESTED** [review]

**Review by palisadoes — APPROVED** [review]

**Pull request merged**